src/components/EventList.jsx

```diff
@@ -29,23 +42,24 @@ const EventsList = props =>
 
 class EventList extends Component {
   render() {
-    let curDateTime = Moment();
-    let upcomingEvents = this.props.events.filter(e =>
-      Moment(e.end).isAfter(curDateTime));
     return (
       <div className="event-list">
-        <h1>upcoming events</h1>
-        {upcomingEvents.length ? (
+        <h1>Upcoming Events</h1>
+        {this.props.upcomingEvents && this.props.upcomingEvents.length ? (
           <div className="fade-in-bottom">
             <table className="scroll">
               <tbody>
-                <EventsList events={upcomingEvents} />
+                <EventsList upcomingEvents={this.props.upcomingEvents} />
               </tbody>
             </table>
           </div>
         ) : (
-          <div className="mb-3">
-            We have more events coming to you soon! Stay tuned!
+          <div className="mt-5 mb-5" id="empty-event-list">
+            <img
```

Comment:
Add illustration for no upcoming events.

Something like:
![after](https://user-images.githubusercontent.com/14525856/81995436-5bb21380-9618-11ea-9cb3-5ef58df1c228.png)